cfme/tests/automate/custom_button/test_infra_objects.py

```diff
@@ -28,6 +30,45 @@
 SUBMIT = ["Submit all", "One by one"]
 
 
+@pytest.fixture(scope="module")
+def cls(appliance):
+    domain = appliance.collections.domains.create(
+        name="domain_{}".format(fauxfactory.gen_alphanumeric(4)), enabled=True
+    )
+    original_class = (
+        domain.parent.instantiate(name="ManageIQ")
+        .namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+    )
+    original_class.copy_to(domain=domain)
+    yield domain.namespaces.instantiate(name="System").classes.instantiate(name="Request")
+    if domain.exists:
+        domain.delete()
+
+
+@pytest.fixture(scope="module")
+def method(cls):
+    meth = cls.methods.create(
+        name="meth_{}".format(fauxfactory.gen_alphanumeric(4)),
+        script=dedent(
+            """
+        # add google url to open
+        vm = $evm.root['vm']
+        $evm.log(:info, "Opening google url")
+        vm.remote_console_url = "https://google.com"
```

Comment:
I don't see a need to actually resolve a domain here, especially one that ends up loading everything google will. The URL doesn't need to resolve or render a page for the test to validate the browser URL, so let's use an IP/domain from the IANA special-use list for the test:

https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml